src/storybook/stand-alone-documentaion/motion/motion.stories.mdx

```diff
@@ -1,13 +1,148 @@
 import { Meta } from "@storybook/addon-docs";
 import { ComponentName } from "../../components";
+import { AnimationOverviews } from "./animation-overviews/animation-overviews";
+import { DurationProductiveTokens } from "./duration-productive-tokens/duration-productive-tokens";
+import { DurationExpressiveTokens } from "./duration-expressive-tokens/duration-expressive-tokens";
+import { DurationTokenTable } from "./duration-token-table/duration-token-table";
+import { EasingOverviews } from "./easing-overviews/easing-overviews";
+import { EasingTokenTable } from "./easing-token-table/easing-token-table";
+import { AnimationTypes } from "./animation-types/animation-types";
+import {
+  EasingDo,
+  EasingDont,
+  ConsistencyDo,
+  ConsistencyDont,
+  SimplicityDo,
+  SimplicityDont,
+  DontDelayDo,
+  DontDelayDont
+} from "./assets";
 import { ComingSoon } from "../../components/coming-soon/coming-soon";
 
-<Meta
-  title="Foundations/Motion"
+<Meta title="Foundations/Motion" />
+
+<ComponentName withFoundationBackground>Motion</ComponentName>
+
+- [Overview](#overview)
```

Comment:
Also there is a difference between the separation in the table of contents to the separation in the page. I don't feel good with insert this page not organized and clear.
My suggestion: Overview, Implementation, Variants, Do's and dont's

Variants will include the animations types (or animation variants)